examples/dynamic_registration/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+debug: true
+network:
+  type: unix
+  address: dynamic-registration-plugin.sock
+settings:
+  read:
+    interval: 5s
+  write:
+    interval: 5s
+  transaction:
+    ttl: 8m
+dynamicRegistration:
+  config:
+    - base: 10
```

Comment:
What is this base thing? A comment free example of what?

Reply:
alone, its comment free, yes. in the context of the greater example, there is comment describing it https://github.com/vapor-ware/synse-sdk/blob/v1.0/examples/dynamic_registration/plugin.go#L49

I could also put that here, but I have found that duplicate descriptions in multiple places means that when something changes, it needs to be updated in all places and often isn't so then there is conflicting information scattered around. 

There are also docs of the new config scheme here (unfortunately not hosted anywhere until pushed to master) https://github.com/vapor-ware/synse-sdk/blob/v1.0/docs/user/configuration.rst

There were numerous places in the design doc for SDK 1.0 (https://docs.google.com/document/d/1BsCgXcS14n9cd8RWMqLK9MC3ntW6UHQjUtl6D3W422s/edit#heading=h.w1aup12h6dmt) where it was said that "auto_enumerate" would be renamed and the scheme changed a bit.